R/generate_price_data.R

```diff
@@ -37,82 +37,38 @@ get_prices <- function(aScenType) {
 
 #' get_hindcast_prices
 #'
-#' @details Read in FAO prices by country and crop,
-#'          Aggregate to GCAM regions and commodities,
+#' @details Read in FAO prices by GCAM regions and commodity. Prices were processed by Ryna.
```

Comment:
It looks like we were able to cut a lot of code here.  Is the idea that a lot of the processing we were formerly doing in this package has now been done outside the package and saved into the input data?

Reply:
Yes. The data I got was pre-processed.